microovn/ovn/environment.go

```diff
@@ -121,15 +134,23 @@ func generateEnvironment(s *state.State) error {
 		}
 
 		server := servers[0]
```

Comment:
It looks to me that there's a quite bit of code overlap between this part of code that generates `{n,s}bInitial` and the code in `connectString()`. What would you think about returning  "initial string" as part of the `connectString()`? Perhaps by changing return value from `string` to some simple `struct` with 2 members?
If not in this PR, maybe we can just open an issue about it.

Reply:
Looking at it, you're right, there's quite a bit of duplicated code and a redundant db transaction as well.

I've went ahead and renamed `connectString` to `environmentString` and modified it to fetch all the data from one transaction, and parse both the connect strings and initial strings afterward. 

I opted to just have the function return two strings for now rather than build out a whole struct, that might be something to consider if this helper continues to grow in scope, or if it ends up that this information gets passed around together often.